docs/content/guides/author-apps/portable-resources/howto-author-portable-resources/index.md

```diff
@@ -0,0 +1,80 @@
+---
+type: docs
+title: "How-To: Author portable resources"
+linkTitle: "Author portable resources"
+description: "Learn how to author portable resources in Radius"
+weight: 200
+categories: "How-To"
+tags: ["portability"]
+---
+
+This guide will teach you how to author a portable resource for your [Radius Application]({{< ref "/guides/author-apps/application/overview" >}}).
+
+## Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius environment]({{< ref "installation#step-3-initialize-radius" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+
+## Step 1: Define a container resource
+
+Create a Bicep file `app.bicep`, add a container resource that will be leveraged by your portable resource later:
+
+{{< rad file="snippets/app.bicep" embed=true marker="//EMPTYCONTAINER" >}}
+
+## Step 2: Define your portable resource
+
+Portable resources provide **abstraction** and **portability** to Radius Applications. Radius currently offers options to manually provision the resources or automatically provision them via Recipes.
+
+{{< tabs Recipe Manual >}}
+
+{{% codetab %}}
```

Comment:
Should there be a code snippet in here?

Reply:
Let me know if the code snippet is alright, tried to include both default and a specified name